.github/workflows/build_and_test.yml

```diff
@@ -555,6 +555,81 @@ jobs:
       with:
         name: test-results-sparkr--${{ inputs.java }}-${{ inputs.hadoop }}-hive2.3
         path: "**/target/test-reports/*.xml"
+        
+  kinesis-asl:
```

Comment:
BTW, do we need to add a new pipeline?

Reply:
Hi @dongjoon-hyun, what do you mean by add a new pipeline? Wonder if you could elaborate a bit more?

Reply:
<img width="466" alt="Screenshot 2023-11-17 at 1 41 58 PM" src="https://github.com/apache/spark/assets/9700541/b43f6062-4127-4734-8c8d-8c1c9cfb2f72">

Reply:
We are extremely careful about the number of jobs because there is a limit. It would be great if this PR can achieve the goal without changing the number of jobs.

Reply:
Appended the tests to an existing pipeline.